{"metadata":{"role":"symbol","modules":[{"name":"SotoCore"}],"roleHeading":"Type Property","fragments":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"waiterFailed","kind":"identifier"},{"text":": ","kind":"text"},{"text":"ClientError","preciseIdentifier":"s:8SotoCore9AWSClientC11ClientErrorV","kind":"typeIdentifier"}],"externalID":"s:8SotoCore9AWSClientC11ClientErrorV12waiterFailedAEvpZ","title":"waiterFailed","symbolKind":"property"},"kind":"symbol","abstract":[{"type":"text","text":"Waiter failed, but without an error. ie a successful api call was an error"}],"hierarchy":{"paths":[["doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClient","doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClient\/ClientError"]]},"sections":[],"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["Linux"],"tokens":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"waiterFailed","kind":"identifier"},{"text":": ","kind":"text"},{"text":"ClientError","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClient\/ClientError","kind":"typeIdentifier","preciseIdentifier":"s:8SotoCore9AWSClientC11ClientErrorV"},{"text":" { ","kind":"text"},{"text":"get","kind":"keyword"},{"text":" }","kind":"text"}]}]}],"schemaVersion":{"patch":0,"major":0,"minor":3},"identifier":{"url":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClient\/ClientError\/waiterFailed","interfaceLanguage":"swift"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/sotocore\/awsclient\/clienterror\/waiterfailed"]}],"references":{"doc://codes.soto.soto-core/documentation/SotoCore/AWSClient/ClientError":{"kind":"symbol","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClient\/ClientError","abstract":[{"type":"text","text":"Errors returned by "},{"code":"AWSClient","type":"codeVoice"},{"type":"text","text":" code"}],"title":"AWSClient.ClientError","type":"topic","role":"symbol","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ClientError","kind":"identifier"}],"navigatorTitle":[{"text":"ClientError","kind":"identifier"}],"url":"\/documentation\/sotocore\/awsclient\/clienterror"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSClient":{"fragments":[{"text":"class","kind":"keyword"},{"text":" ","kind":"text"},{"text":"AWSClient","kind":"identifier"}],"type":"topic","url":"\/documentation\/sotocore\/awsclient","role":"symbol","abstract":[{"text":"Client managing communication with AWS services","type":"text"}],"kind":"symbol","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClient","title":"AWSClient","navigatorTitle":[{"text":"AWSClient","kind":"identifier"}]},"doc://codes.soto.soto-core/documentation/SotoCore/AWSClient/ClientError/waiterFailed":{"kind":"symbol","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClient\/ClientError\/waiterFailed","abstract":[{"type":"text","text":"Waiter failed, but without an error. ie a successful api call was an error"}],"title":"waiterFailed","type":"topic","role":"symbol","fragments":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"waiterFailed","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:8SotoCore9AWSClientC11ClientErrorV","text":"ClientError","kind":"typeIdentifier"}],"url":"\/documentation\/sotocore\/awsclient\/clienterror\/waiterfailed"},"doc://codes.soto.soto-core/documentation/SotoCore":{"type":"topic","url":"\/documentation\/sotocore","role":"collection","abstract":[{"text":"The core framework for Soto, a Swift SDK for AWS","type":"text"}],"kind":"symbol","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","title":"SotoCore"}}}